### ISSUE TITLE
`rocminfo` deb package has wrong (or no) dependencies

### ISSUE THREAD

**Issue by baryluk** [ROCm/ROCm#1129]
```
$ apt show rocminfo
Package: rocminfo
Version: 1.30500.0
Priority: optional
Section: devel
Maintainer: Advanced Micro Devices Inc.
Installed-Size: 76.8 kB
Download-Size: 25.1 kB
APT-Manual-Installed: no
APT-Sources: http://repo.radeon.com/rocm/apt/debian xenial/main amd64 Packages
Description: Radeon Open Compute (ROCm) Runtime rocminfo tool
```

this is simply wrong.

You are missing dependency on `libstdc++6:amd64` and `hsa-rocr-dev`. It is also very unfortunate that the `/opt/rocm-3.5.0/lib/libhsa-runtime64.so.1` is in `hsa-rocr-dev` and not in the `libhsa1` with headers (and any potential static libraries in the future) in `libhsa-dev`.

**Comment by ye-luo** [comment]
Also found it very broken.
```
$ rocminfo
rocminfo: error while loading shared libraries: libhsakmt.so.1: cannot open shared object file: No such file or directory
```
I cannot find  libhsakmt.so.1 under /opt/rocm
but hsakmt-roct is shown installed.
After forcing a reinstall, the problem goes away.
```
sudo apt reinstall hsakmt-roct
```

**Comment by baryluk** [comment]
Still broken in 3.8.0, just like it was also broken in 3.7.0 and 3.5.0.

**Comment by ROCmSupport** [comment]
Thanks @baryluk for reaching out.
I will take a look and update soon on this.
Thank you.

**Comment by bugparty** [comment]
I tried in rocm4.1, `sudo apt reinstall hsakmt-roct` works

**Comment by baryluk** [comment]
Still broken in rocm 4.1, same way as before.

**Comment by ROCmSupport** [comment]
Update:
Assigned to dev and he is looking into it.
I will keep you posted.
Thank you.

**Comment by ROCmSupport** [comment]
Fix is ready and changes are pushed to internal code and fix is available most likely in 4.3 or less likely in 4.4 code.
Thank you.

**Comment by baryluk** [comment]
Still broken in rocm 4.3. Hopefully it got fixed in 4.4.

**Comment by keryell** [comment]
@baryluk I guess if it is not already fixed yet you can send a PR. It might be more efficient than opening an issue... :-/

**Comment by nartmada** [comment]
Hi @baryluk, please check latest ROCm Documentation and ROCm 5.7.1 to see if your issue has been resolved.  If resolved, please close the ticket.  Thanks.

**Comment by baryluk** [comment]
@nartmada Unfortunately ROCm 5.7.1 cannot be installed (the instructions to install ROCm 5.7.1 mention unversioned ROCm repo, so it actually tries to install 6.0.0), plus it breaks anyway, as it is referencing some old Python versions.

I cannot install ROCm 6.0.0 either, because install instructions are broken, and apt is getting 404 from web servers hosting repos when following install instructions.

**Comment by nartmada** [comment]
Hi @baryluk, thanks for getting back.  Let me talk to the internal team for next steps on this ticket.

**Comment by nartmada** [comment]
Hi @baryluk, thank you for your patience.  Can you please try updating the Ubuntu version and try installing ROCm 6.0 again?  Please include info on any installation error encountered.  Thanks.

**Comment by nartmada** [comment]
Hi @baryluk, can you please provide the CPU, GPU, and Operating System info?  We will try to repro your issue here at AMD using ROCm 5.7.1.  Thanks.

**Comment by nartmada** [comment]
Hi @baryluk, can you please try ROCm 6.0.2?  An internal developer has looked over the rocminfo package for 6.0.2 and the dependency appear to be correct. Thanks.

**Comment by baryluk** [comment]
Let me try

**Comment by baryluk** [comment]
So, now it is:

`Depends: hsa-rocr, kmod, pciutils, rocm-core`

It is now way better than what it was originally (initially it was empty). It does technically work.

Still, technically stdc++ is not properly mentioned.

Nor is hsa-rocr has any version constraints.

Here are deps graph for the main binary:

![image](https://github.com/ROCm/ROCm/assets/110322/a2daab33-707e-4999-8e09-26f378131944)

Also it missed dependency on `python3`, used by another script in `rocminfo` package.

Adding something like this:

`libc6 (>= 2.34), libgcc-s1 (>= 3.0), libstdc++6 (>= 11), python3`

should help, but of course exact libc6 and stdc++6 version depends on a build environment.

**Comment by nartmada** [comment]
Thank you @baryluk.  I will pass your feedback to internal team.

**Comment by ppanchad-amd** [comment]
@baryluk This is fixed in the latest ROCm 6.2. Thanks!
